src/coprocessor/dag/rpn_expr/impl_arithmetic.rs

```diff
@@ -313,6 +313,81 @@ impl ArithmeticOp for DecimalMultiply {
     }
 }
 
+#[derive(Debug)]
+pub struct IntDivideInt;
+
+impl ArithmeticOp for IntDivideInt {
```

Comment:
According to the implementation of `int_divide_int`, it first checks `rhs == 0`. If so, it returns `Ok(None)`.